core/tests.py

```diff
@@ -0,0 +1,15 @@
+# coding: utf-8
+from django.test import TestCase
+
+
+class AboutpageTest(TestCase):
+    def setUp(self):
+        self.resp = self.client.get('/about/')
+
+    def test_get(self):
+        'GET / must return status code 200'
```

Comment:
1) That docstring is not beautiful as it could be.
2) Document a test is a "weird" thing because the test itself is already a documentation of how your code works.

Just my two cents here :)